aten/src/ATen/xpu/detail/XPUHooks.cpp

```diff
@@ -98,6 +98,19 @@ DeviceIndex XPUHooks::getCurrentDevice() const {
   return at::xpu::current_device();
 }
 
+c10::Stream XPUHooks::getCurrentStream(DeviceIndex device) const {
+  return at::xpu::getCurrentXPUStream(device);
+}
+
+c10::Stream XPUHooks::getDefaultStream(DeviceIndex device) const {
+  return at::xpu::getDefaultXPUStream(device);
```

Comment:
Default stream has an implicit synchronization for CUDA. Even though the streams are created with `noblocking` flag in PyTorch, default stream still blocks the other stream created in external lib. I don't think every kind of accelerator supports the implicit synchronization. At least XPU doesn't have a counterpart.